data/ggm0001/ggm001/__cts__.xml

```diff
@@ -0,0 +1,9 @@
+<ti:work xmlns:ti="http://chs.harvard.edu/xmlns/cts" groupUrn="urn:cts:greekLit:ggm0001" xml:lang="grc" urn="urn:cts:greekLit:ggm0001.ggm001">
+  <ti:title xml:lang="lat">Anametresis Pontou</ti:title>
+  <ti:edition urn="urn:cts:greekLit:ggm0001.ggm001.1st1K-grc1" workUrn="urn:cts:greekLit:ggm0001.ggm001">
+    <ti:label xml:lang="lat">
+      Anametresis Pontou
+    </ti:label>
+    <ti:description xml:lang="eng">Anonymous, creator; Geographi graeci minores Volumen Primum, ed. Karl Müller; Ambroise Firmin Didot, Paris, 1855</ti:description>
```

Comment:
The xml:lang for an element that has both English and Latin (or any other combination of more than one language) should be "mul" (for multiple languages). I think that is the case on several of your `__cts__.xml` files here.

Reply:
should I change the xml:lang in the ti:work element only, or in the whole file?

Reply:
You only need to change it on the `<ti:description>` element. This is because it contains both English (e.g., "creator") and Latin ("Geographi graeci minores Volumen Primum"). The rest of the tags, both on this file and the other `__cts__.xml` files can remain the same.